website/static/js/pages/project-dashboard-page.js

```diff
@@ -285,4 +286,19 @@ $(document).ready(function () {
     $('a.btn').mouseup(function(){
         $(this).blur();
     });
+
+    if (window.contextVars.node.isPublic) {
+        m.mount(document.getElementById('shareDropdown'),
+                m.component(SocialShare.ShareDropdown,
+                    {file_name: window.contextVars.node.title, share_url: window.location.href}));
+
+        // Make sure the drop down share menu is not obscured
```

Comment:
I've just made a fix on develop that should make these two event handlers unnecessary. The tooltip should properly disappear when hovering off of the "Duplicate" button.

@acshi Can you verify this after you pull from develop?